### PR TITLE
Because apparently this got passed 3 people...

### PR DESCRIPTION
**Pull request by NickLaMuro**
Evidence:

- https://github.com/chrisarcand/miqversions/pull/10
- https://github.com/chrisarcand/miqversions/pull/13

And off course, we can't forget our merging overlord, Chris...

(just be happy that I didn't "at you" in the commit message...)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ $ miqversions
 | Fine           | 5.8.z                        | 4.5        | 2.3.x | 9.5.x      |
 | Gaprindashvili | 5.9.z                        | 4.6        | 2.3.x | 9.5.x      |
 | Hammer         | 5.10.z                       | 4.7        | 2.4.x | 9.5.x      |
-| Ivanchuck      | 5.11.z                       | 4.8        | 2.4.x | 9.5.x      |
```
Comment (jrafanie) [inline README.md:23]: ???  It's spell wrong.  Why change the PG version?  What am I missing?

Comment (jrafanie) [reply README.md:23]: my brain isn't working... I thought this was already fixed... see https://github.com/chrisarcand/miqversions/pull/14 (no "chuck" typo and pg version is 10)

Comment (NickLaMuro) [reply README.md:23]: yeah, I guess everything here was copy-pasta and kinda borked... fixed

**Comment by juliancheal** [comment]
LGTM 👍

**Pull request merged**